packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/DoneAndTalkButton/DoneAndTalkButtonContainer.js

```diff
@@ -31,14 +31,14 @@ class DoneAndTalkButtonContainer extends React.Component {
       subject
     } = this.props
 
-    if (shouldWeShowDoneAndTalkButton && subject.id) {
-      function openTalkLinkAndClick (event) {
-        const isCmdClick = event.metaKey
+    function openTalkLinkAndClick (event) {
```

Comment:
This should probably be a class method, rather than being defined in the render method